### ISSUE TITLE
Feature request: Override TB close button -> minimize to tray

### ISSUE THREAD

**Issue by geckolinux** [gyunaev/birdtray#13]
Hi there, first of all, thanks a *million* for this utility, it's really saving my workflow after losing the Firetray extension in TB 60.0+.

With Firetray, I could make it override the TB close button to minimize it to the tray. This is now undoable muscle memory for me, and I constantly close TB when I just want it to go the tray. Is there any way to add this functionality to Birdtray?

Thanks again!

**Comment by gyunaev** [comment]
No, this is not possible unfortunately. However I added an option to restart closed Thunderbird; see 601577b
